# Message for Velocity motor and Control turn
int32 turn
int32 vel